internal/csi-addons/rbd/replication.go

```diff
@@ -270,24 +272,26 @@ func (rs *ReplicationServer) EnableVolumeReplication(ctx context.Context,
 	}
 	defer rs.VolumeLocks.Release(volumeID)
 
-	rbdVol, err := corerbd.GenVolFromVolID(ctx, volumeID, cr, req.GetSecrets())
-	defer func() {
-		if rbdVol != nil {
-			rbdVol.Destroy(ctx)
-		}
-	}()
+	mgr := rbd.NewManager(req.GetParameters(), req.GetSecrets())
```

Comment:
note that the CSI-instance id (drivername) needs to be passed to the manager in an upcoming change.